### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by infinisil**
Same as Nixpkgs (MIT), but with me as the copyright holder, since effectively all code has been written by me

### CODE REVIEW

File: LICENSE
Comment (philiptaron) [inline LICENSE:1]:
```
> be me
> reading PR
> see email, wonder what website is like
> go to infinisil.com
```

# Infinisil's Website
It's empty!

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) 2023-2024 Silvan Mosberger <contact@infinisil.com>
```
Comment (philiptaron) [inline LICENSE:3]:
```suggestion
Copyright (c) 2023-2024 Silvan Mosberger <contact@infinisil.com> and contributors
```

https://github.com/NixOS/nixpkgs/commit/463793827f5ef7afc11f6a83501332830ed76078

**Review by philiptaron — CHANGES_REQUESTED** [review]

File: LICENSE
Comment (infinisil) [reply LICENSE:1]: No code is best code! :laughing:

File: LICENSE (hunk 1 above)
Comment (infinisil) [reply LICENSE:3]: Fair!

**Review by philiptaron — APPROVED** [review]

**Pull request merged**